cppcache/integration/test/FunctionExecutionTest.cpp

```diff
@@ -324,7 +324,9 @@ TEST(FunctionExecutionTest, testThatFunctionExecutionThrowsExceptionNonHA) {
       .withType("PARTITION")
       .execute();
 
-  cluster.getServers()[2].stop();
+  auto &targetServer = cluster.getServers()[2];
+  targetServer.stop();
+  targetServer.wait();
```

Comment:
Why this separation of concerns between stop() and wait()?  Is there ever a circumstance where we call stop() and don't intend for the process to actually be terminated yet when the function returns?